### ISSUE TITLE
Segment Fault after Import Tensorflow

### ISSUE THREAD

**Issue by zjhzxhz** [tensorflow/tensorflow#2636]
GitHub issues are for bugs / installation problems / feature requests.  
For general support from the community, see [StackOverflow](https://stackoverflow.com/questions/tagged/tensorflow).
To make bugs and feature requests more easy to find and organize, we close issues that are deemed
out of scope for GitHub Issues and point people to StackOverflow.

For bugs or installation issues, please provide the following information.
The more information you provide, the more easily we will be able to offer
help and advice.

### Environment info
Operating System: Debian 8.5

Installed version of CUDA and cuDNN:  No
(please attach the output of `ls -l /path/to/cuda/lib/libcud*`):

If installed from binary pip package, provide:

1. Which pip package you installed: Ubuntu/Linux 64-bit, CPU only
2. The output from python -c "import tensorflow; print(tensorflow.__version__)": Segment Fault

If installed from sources, provide the commit hash:

### Steps to reproduce
1. Run Python with `python`
2. Import Tensorflow `import tensorflow`
3. Got error `Segment Fault`

### What have you tried?

N/a

### Logs or other output that would be helpful
(If logs are large, please upload as attachment).

![screenshot from 2016-06-03 15 51 26](https://cloud.githubusercontent.com/assets/1730504/15772309/fde464e6-29a2-11e6-91bf-0428672c8b52.png)

**Comment by aselle** [comment]
Could you show uname -a on the machine and also let us know what PIP binary package you installed or what sha hash if you installed from source.

**Comment by wookayin** [comment]
I guess #2034 might be relevant

**Comment by zjhzxhz** [comment]
Here's the result of `uname -a`:

```
hzxie@XieHaozhe-PC:~$ uname -a
Linux XieHaozhe-PC 3.16.0-4-amd64 #1 SMP Debian 3.16.7-ckt25-2 (2016-04-08) x86_64 GNU/Linux
```

And the output of installed packages are listed below:

```
hzxie@XieHaozhe-PC:~$ pip list
cffi (0.8.6)
chardet (2.3.0)
cryptography (0.6.1)
defusedxml (0.4.1)
docutils (0.12)
lxml (3.4.0)
nose (1.3.7)
nose-parameterized (0.5.0)
numpy (1.11.0)
pexpect (3.2)
Pillow (2.6.1)
pip (8.1.2)
ply (3.4)
protobuf (3.0.0b2)
pycparser (2.10)
pycups (1.9.63)
pycurl (7.19.5)
Pygments (2.0.1)
pygobject (3.14.0)
pyOpenSSL (0.14)
pysmbc (1.0.15.3)
python-apt (0.9.3.12)
python-debian (0.1.27)
python-debianbts (1.11)
pyxdg (0.25)
reportbug (6.6.3)
reportlab (3.1.8)
requests (2.10.0)
roman (2.0.0)
scikit-learn (0.17.1)
scipy (0.17.0)
setuptools (22.0.0)
six (1.10.0)
SOAPpy (0.12.22)
socketIO-client (0.6.5)
tensorflow (0.8.0)
Theano (0.8.2)
websocket-client (0.37.0)
wheel (0.29.0)
wstools (0.4.3)
```

**Comment by girving** [comment]
@zjhzxhz: Can you check if doing `import numpy` before `import tensorflow` fixes the problem, as per #2034.  Alternatively, you could try upgrading to 0.9, which shouldn't have this problem.

**Comment by aselle** [comment]
Automatically closing due to lack of recent activity. Please reopen when further information becomes available.
